jax/_src/test_util.py

```diff
@@ -419,6 +419,12 @@ def is_device_tpu(version: int | None = None, variant: str = "") -> bool:
     return "v5 lite" in device_kind
   return expected_version in device_kind
 
+def is_device_gpu_at_least(capability: str) -> bool:
```

Comment:
Maybe `is_cuda_compute_capability_at_least` ?

Is the intent that this test only does something for NVIDIA GPUs? If so, I'd probably put "CUDA" in the name.

What should happen on an AMD GPU here?

Reply:
Ooops, sorry. Done in #21137.